### ISSUE TITLE
Update the TaxTweb to GEM Taxonomy v3

### ISSUE THREAD

**Issue by raoanirudh** [gem/oq-platform-taxtweb#28]
The TaxTweb is a very useful tool for obtaining validated short taxonomy strings for building types. The current string outputs, however, are still based on the older GEM Taxonomy v2. The current version of the GEM Taxonomy, v3.x / Multi-hazard Taxonomy, was released in March 2018. While most of the attributes remain unchanged, there are a few changes. It would be really helpful if the TaxTweb could be updated to support the new GEM Taxonomy.

cc: @CatalinaYepes

**Comment by pslh** [comment]
Note that this implies changes also to the Taxonomy glossary, the IDCT data collection tool and possibly to other software tools (ipython notebooks, risk toolkit?).

**Comment by CatalinaYepes** [comment]
I think we should assign some time to do this. The models in the GRM are using the latest version of the taxonomy. The document generated for GED4ALL is the starting point to do the modifications and add the additional descriptions to the Taxonomy glossary. I don't think at this point we need to modify IDCT (there are other type of improvements that have not been done and have higher priority).

**Comment by pslh** [comment]
See also [https://github.com/gem/oq-platform2/issues/89](https://github.com/gem/oq-platform2/issues/89)

**Comment by nicolepaul** [comment]
I agree with the others on the Risk team that it would be worthwhile to assign some time to this. Additionally, we could improve our unit tests / create utility functions for exposure modellers to conform to GED4ALL if this work is completed. At the moment, our exposure models are inconsistent.

**Comment by raoanirudh** [comment]
See also https://github.com/gem/global_vulnerability_model/issues/3
